recipes/setuptools-rust/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "setuptools-rust" %}
+{% set version = "0.10.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ef596edebe2d075cd76e40e8c1f18ac54f14b93dbccef48cbf91d584372b4907
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - semantic_version >=2.6.0
+    - setuptools
+    - toml >=0.9.0
+
+test:
+  imports:
+    - setuptools_rust
+
+about:
+  home: https://github.com/PyO3/setuptools-rust
+  license: MIT
+  license_family: MIT
```

Comment:
Please add the ``license_file``

Reply:
There is an open issue for that

https://github.com/PyO3/setuptools-rust/issues/40

Reply:
As this is listed by the author of the project in pypi as MIT, I think we can package the license along with this recipe and it should be fine.

https://pypi.org/project/setuptools-rust/0.10.3/

Reply:
I agree

Reply:
Related to
https://github.com/PyO3/setuptools-rust/pull/41